pkg/osquery/table/touchid_user_darwin.go

```diff
@@ -0,0 +1,168 @@
+package table
+
+import (
+	"bytes"
+	"context"
+	"io/ioutil"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"syscall"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/pkg/errors"
+
+	"github.com/kolide/osquery-go"
+	"github.com/kolide/osquery-go/plugin/table"
+)
+
+func TouchIDUserConfig(client *osquery.ExtensionManagerClient, logger log.Logger) *table.Plugin {
+	t := &touchIDUserConfigTable{
+		client: client,
+		logger: logger,
+	}
+	columns := []table.ColumnDefinition{
+		table.IntegerColumn("uid"),
+		table.IntegerColumn("fingerprints_registered"),
+		table.IntegerColumn("touchid_unlock"),
+		table.IntegerColumn("touchid_applepay"),
+		table.IntegerColumn("effective_unlock"),
+		table.IntegerColumn("effective_applepay"),
+	}
+
+	return table.NewPlugin("touchid_user_config", columns, t.generate)
```

Comment:
Please prepend all table names with `kolide_` so we don't accidentally conflict with other plugins